web/components/BabyBook/BabyModal.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+import TextInput from "@components/atoms/TextInput";
+
+
+interface Props {
+    image: string;
+    description?: string;
+    edit: true | false
+    deselect?: () => void;
+    selectImage?: (
+      arg0: number,
+      arg1: number,
+      arg2: number,
+      arg3: boolean
+    ) => void;
+    currentIndexs?: {
+      i: number;
+      j: number;
+      k: number;
+    };
+  }
+
+export default function BabyModal({ image, description, edit}: Props) {
+
+    const [newDescription, setNewDescription] = useState("");
+
+    return (
+        <div className="flex flex-col w-full h-full items-center sm:flex-row">
+            <div className={`relative w-full bg-image-gray sm:w-[55%] sm:h-full ${edit ? "flex justify-center items-center" : "flex justify-center h-[65%] sm:items-center"}`}>
+                <img
+                    src={image}
+                    className={`sm:w-[21rem] sm:max-h-[30rem] ${edit ? "max-h-[22rem]" : "w-4/5 object-cover"}`}
+                ></img>
+                {edit && (
+                    <button className="w-[7.563rem] h-[1.688rem] bg-black/70 rounded-[999px] absolute bottom-3 right-3 sm:bottom-8 sm:right-[10.5rem]">
```

Comment:
This button looks great which I know is hard to get right since it's layered on top of the image :D